src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssDeclarations = sourceString.split(';');
+  const styleObject = {};
+
+  cssDeclarations.forEach((property) => {
+    const propertyArr = property.split(':', 2);
+    const [propertyName, propertyValue] = propertyArr;
+
+    if (propertyValue) {
```

Comment:
Checking for the existence of the `propertyName` value to avoid empty keys in the object.
```suggestion
    if (propertyName && propertyValue) {
```